src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -74,24 +75,21 @@ public MethodSignatureEncoder PropertySignature(bool isInstanceProperty = false)
             return new MethodSignatureEncoder(Builder, hasVarArgs: false);
         }
 
-        public void CustomAttributeSignature(out FixedArgumentsEncoder fixedArguments, out CustomAttributeNamedArgumentsEncoder namedArguments)
+        [return: TupleElementNames(new[] { "FixedArguments", "NamedArguments" })]
```

Comment:
So far it looks correct. 
For tuples with length under 8 it is very likely that this will be the encoding.